examples/fargate/cloudwatchlogs/permissions.json

```diff
@@ -6,7 +6,9 @@
 			"logs:CreateLogStream",
 			"logs:CreateLogGroup",
 			"logs:DescribeLogStreams",
-			"logs:PutLogEvents"
+			"logs:PutLogEvents",
+			"logs:PutRetentionPolicy",
+                        "logs:DeleteRetentionPolicy"
```

Comment:
Why delete?

Reply:
Because, If I want remove the retentionPolicy of my logGroup, I can do it:

![image](https://github.com/aws-samples/amazon-eks-fluent-logging-examples/assets/87954654/46535b90-2b82-4f3a-86bd-7f884761f4a5)